### PR TITLE
Upgrade Elasticsearch to 7.9.0

### LINKED ISSUE #626: Search performance issue when using Elasticsearch [7.7.0, 7.8.x]
**Describe the bug**
Significant performance drop in searches/aggregations when using Elasticsearch 7.7.0+ in Snow Owl 7.7.0+ release.

**To Reproduce**

Run a validation rule that uses either a search or aggregation with a large number of terms filter values (all active concept IDs listed, huge ECL/QL result set, etc.).

**Expected behavior**
Running any search with large number of term filter values should not cause significant performance drop in computation time.

**Version**
 - Snow Owl Version: 7.7.0, 7.8.0
 - Elasticsearch Version (external): 7.7.0, 7.8.0

**Additional context**
Elasticsearch PR where the Percolator FetchPhase implementation rewrite happened:
https://github.com/elastic/elasticsearch/pull/53728
Lucene 8.6.0 PR where the potential fix has been introduced:
https://github.com/apache/lucene-solr/pull/1465
Related Elastic.co Discussion:
https://discuss.elastic.co/t/performance-problem-in-percolator-fetch-phase-classes/243748

This performance issue is going to be fixed when Elasticsearch 7.9.0 releases (it uses Lucene 8.6.0 under the hood).

### PR DESCRIPTION
**Pull request by cmark**
Fixes #626

### CODE REVIEW

File: commons/com.b2international.index/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -57,57 +57,57 @@ Import-Package: javax.management,
...
+ lib/elasticsearch-core-7.9.0.jar,
+ lib/elasticsearch-geo-7.9.0.jar,
+ lib/elasticsearch-rest-client-7.9.0.jar,
+ lib/elasticsearch-rest-high-level-client-7.9.0.jar,
+ lib/elasticsearch-secure-sm-7.9.0.jar,
+ lib/elasticsearch-ssl-config-7.9.0.jar,
+ lib/elasticsearch-x-content-7.9.0.jar,
  lib/hppc-0.8.1.jar,
  lib/hppcrt-0.7.5.jar,
  lib/jna-4.5.1.jar,
```
Comment (apeteri) [inline commons/com.b2international.index/META-INF/MANIFEST.MF:80]: JNA is at 5.5.0 in the distribution .zip file.

**Review by apeteri — APPROVED** [review]
👍

**Comment by apeteri** [comment]
It might be worth checking their release .zip for further version differences, I couldn't find anything outside the JNA jar with a quick look-through.

**Comment by cmark** [comment]
Already did and it looked good to me. 😃

**Pull request merged**